transaction.go

```diff
@@ -60,6 +60,9 @@ type Operation struct {
 
 	// The limit (update, delete).
 	Limit int
+
+	// The filters of the array inside the documents (update).
```

Comment:
I would suggest `// The array filter conditions (update).`.

Reply:
Done.